src/lib/crypto/kimchi_bindings/wasm/Cargo.toml

```diff
@@ -23,9 +23,7 @@ libc = "0.2.62"
 num-bigint = { version = "0.4.0" }
 once_cell = "1.10.0"
 paste = "1.0.5"
-# Version for proc-macro2 needs to be 1.0.60+ to be compatible with newer rust versions
-# https://github.com/rust-lang/rust/issues/113152
-proc-macro2 = { version = "=1.0.66", features = ["default", "proc-macro"] }
+proc-macro2 = { version = "1.0.94", features = ["default", "proc-macro"] }
```

Comment:
This will conflict with https://github.com/MinaProtocol/mina/pull/16797/files. I would suggest to merge first 16797 and I will take care of updating this PR accordingly.

Reply:
Should be ok now.